### PR TITLE
feat: add config for trading mode metric

### PR DESCRIPTION
**Pull request by baroooo**

### CODE REVIEW

File: config.local.yaml [hunk 1]
```diff
@@ -71,3 +74,18 @@ metrics:
...
+    variants:
+      - ['CELOUSD']
+      - ['CELOEUR']
+      - ['CELOBRL']
+      - ['CELOXOF']
+      - ['USDCUSD']
+      - ['USDCEUR']
+      - ['USDCBRL']
+      - ['EUROCEUR']
+      - ['XOFEUR']
```
Comment (philbow61) [inline config.local.yaml:90]: i think this needs to bee EURXOF

File: config.yaml [hunk 2]
```diff
@@ -71,3 +74,18 @@ metrics:
...
+    variants:
+      - ['CELOUSD']
+      - ['CELOEUR']
+      - ['CELOBRL']
+      - ['CELOXOF']
+      - ['USDCUSD']
+      - ['USDCEUR']
+      - ['USDCBRL']
+      - ['EUROCEUR']
+      - ['XOFEUR']
```
Comment (philbow61) [inline config.yaml:90]: same here

**Review by ninocomputer — APPROVED** [review]

**Review by philbow61 — CHANGES_REQUESTED** [review]

File: config.local.yaml (hunk 1 above)
Comment (ninocomputer) [reply config.local.yaml:90]: we have both directions

Comment (baroooo) [reply config.local.yaml:90]: hmm I think this refers to the variable name at line 7 and not the actual name of the rate feed.

I can change the name for consistency but that would also affect the label names in grafana, causing a weird gap in all metrics

image

**Review by philbow61 — APPROVED** [review]
LGTM 👍

**Pull request merged**